scripts/make_gitrev.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+file=include/spl/sys/zfs_gitrev.h
+
+#
+# Set default file contents in case we bail.
+#
+rm -f $file
+echo "#define\tZFS_META_GITREV \"unknown\"" >>$file
```

Comment:
s/echo/echo -e/

```
       -e     enable interpretation of backslash escapes
```

Reply:
Thanks, that explains the build failures!

Reply:
The `printf` built in should work too as an alternative.

Reply:
printf isn't a builtin for sh, but it does usually exist as /usr/bin/printf